HW7/Mengyuan_HW7.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+import numpy as np
+import pandas as pd 
+import matplotlib.pyplot as plt
+import folium 
+from folium import plugins
+import webbrowser
+
+
+# Set global settings and macros.
+MAX_SHOW =10000
+HOUSE_URL = 'houses.html'
+HOUSE_HEAT_URL = "househeatmap.html"
+CRIME_HEAT_URL = "crimeheatmap.html"
+pd.set_option('display.max_columns', None) # To display all columns
+
+
+# Read in king county house data.
+data = pd.read_csv("../../Data/kc_house_data.csv", parse_dates = ['date'])
```

Comment:
Remove whitespace in rename parse_date = ['date'] to parse_date=['date'] ...
Similar arguments should also remove whitespace